OpenEdXMobile/src/main/java/org/edx/mobile/viewModel/VideoViewModel.kt

```diff
@@ -0,0 +1,25 @@
+package org.edx.mobile.viewModel
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+
+@HiltViewModel
+class VideoViewModel @Inject constructor() : ViewModel() {
+
+    private val _clearChoices = MutableLiveData(false)
+    val clearChoices: LiveData<Boolean> = _clearChoices
+
+    private val _selectedVideosPosition = MutableLiveData(-1)
```

Comment:
same here

Reply:
It will be handled automatically once we move to Event Observer. Changing the type to Long causes issues in ListView and requires further changes.